lib/active_resource/active_job_serializer.rb

```diff
@@ -0,0 +1,25 @@
+module ActiveResource
+  class ActiveJobSerializer < ActiveJob::Serializers::ObjectSerializer
+    def serialize(resource)
+      super(
+        "class" => resource.class.name,
+        "persisted" => resource.persisted?,
+        "prefix_options" => resource.prefix_options.as_json,
+        "attributes" => resource.attributes.as_json
+      )
+    end
+
+    def deserialize(hash)
+      hash["class"].constantize.new(hash["attributes"]).tap do |resource|
+        resource.persisted      = hash["persisted"]
+        resource.prefix_options = hash["prefix_options"]
+      end
+    end
+
+    private
+
+      def klass
+        ActiveResource::Base
+      end
+  end
+end if defined?(ActiveJob::Serializers::ObjectSerializer)
```

Comment:
Think it's partially odd that we require a file that then ends up not defining anything. Not sure if there's precedent for us handling a case like that in this way before 😊